tensorflow_transform/mappers.py

```diff
@@ -394,7 +395,8 @@ def _to_tfidf(term_frequency, reduced_term_freq, corpus_size, smooth):
         tf.to_double(reduced_term_freq))) + 1
 
   gathered_idfs = tf.gather(tf.squeeze(idf), term_frequency.indices[:, 1])
-  tfidf_values = tf.to_float(term_frequency.values) * tf.to_float(gathered_idfs)
+  tfidf_values = tf.dtypes.cast(term_frequency.values, 'float64', name='Calc_tfidf1') *\
```

Comment:
To be consistent with our style guide, let's change the backslash line continuation to parentheses:

```
  tfidf_values = (
      tf.cast(term_frequency.values, tf.float64) *
      tf.cast(gathered_idfs, tf.float64))
```